src/sockperf.cpp

```diff
@@ -2349,13 +2364,19 @@ void cleanup() {
                 }
                 if (s_user_params.addr.ss_family == AF_UNIX) {
                     unlink(s_user_params.client_bind_info.addr_un.sun_path);
+#ifndef WIN32 // AF_UNIX with DGRAM isn't supported in Win32
                     if (s_user_params.mode == MODE_CLIENT && s_user_params.sock_type == SOCK_DGRAM) { // unlink binded client
                         std::string sun_path = build_client_socket_name(&s_user_params.addr.addr_un, getpid(), ifd);
                         log_dbg("unlinking %s", sun_path.c_str());
                         unlink(sun_path.c_str());
                     }
+#endif
                     if (s_user_params.mode == MODE_SERVER)
+#ifdef WIN32
+                        DeleteFileA(g_fds_array[ifd]->server_addr.addr_un.sun_path);
+#else
                         unlink(g_fds_array[ifd]->server_addr.addr_un.sun_path);
```

Comment:
move `DeleteFileA` / `unlink` into os_abstract.
consider using `remove()` instead of `DeleteFileA`